src/deterministic.jl

```diff
@@ -18,26 +18,64 @@ function Deterministic(
     data::Union{Dict{Dates.DateTime, Any}, SortedDict{Dates.DateTime, Any}};
     normalization_factor::NormalizationFactor = 1.0,
     scaling_factor_multiplier::Union{Nothing, Function} = nothing,
-    timestamp = :timestamp,
     resolution::Union{Dates.Period, Nothing} = nothing,
 )
     for (k, v) in data
         if v isa DataFrames.DataFrame
-            data[k] = TimeSeries.TimeArray(v; timestamp = timestamp)
+            data[k] = Array(v)
```

Comment:
What happens if the dataframe has a column containing timestamps?

Reply:
it's irrelevant in this case. The DataFrame represents a window of the forecast.

Reply:
Then shouldn't you throw an error if the number of columns is greater than 1?